languages/julia/concepts/conditionals/about.md

```diff
@@ -0,0 +1,4 @@
+There are three primary conditional statements that are used in Julia:
+- [`if`-statements](../if-statements/about.md)
+- [The ternary operator](../ternary-operator/about.md)
+- [Short-circuit evaluation](../short-circuit-evaluation/about.md)
```

Comment:
```suggestion
- [`if`-statements](https://exercism.lol/tracks/julia/concepts/if-statements/)
- [The ternary operator](https://exercism.lol/tracks/julia/concepts/ternary-operator/)
- [Short-circuit evaluation](https://exercism.lol/tracks/julia/concepts/short-circuit-evaluation/)
```

See https://exercism-team.slack.com/archives/CR91YFNG3/p1606168123251500?thread_ts=1606166562.227300&cid=CR91YFNG3 and above

Reply:
No trailing slashes

Reply:
Thanks